cms/jinja2/templates/pages/release_calendar/release_calendar_page.html

```diff
@@ -1,54 +1,75 @@
-                <div class="ons-grid__col ons-col-10@m ">
-                    <h1 class="ons-u-fs-3xl common-header__heading">
-                        {{ page.title }}
-                    </h1>
-                </div>
-                <div class="ons-grid__col ons-col-2@m common-header__kitemark">
-                    {% if page.is_accredited %}
-                        {% with accredited_logo_classes="common-header__accreditation-logo" %}
-                            {% include "templates/components/accredited/accredited-logo.html" %}
-                        {% endwith %}
-                    {% endif %}
-                </div>
-            </div>
-        </div>
+    {% set hero_topic = _("Release") %}
+    {% set breadcrumbs_aria_label = _("Breadcrumbs") %}
+
+    {% if page.release_date or page.release_text %}
+        {% set release_date_term = _("Release date:") %}
+        {# fmt:off #}
+        {% set release_date_html =
+            {
+                "term": release_date_term,
+                "descriptions": [
+                    {
+                        "description": page.release_date|date("DATE_FORMAT") or page.release_date_text
+                    }
+                ]
+            }
+        %}
+        {# fmt:on #}
+    {% else %}
+        {% set release_date_html = "" %}
+    {% endif %}
+
+    {% if page.next_release_date or page.next_release_text %}
+        {% set next_release_date_term = _("Next release:") %}
+        {# fmt:off #}
+        {% set next_release_html =
+            {
+                "term": next_release_date_term,
+                "descriptions": [
+                    {
+                        "description": page.next_release_date|date("DATE_FORMAT") or page.next_release_text
+                    }
+                ]
+            }
+        %}
+        {# fmt:on #}
+    {% else %}
+        {% set next_release_html = "" %}
+    {% endif %}
+
+    {# fmt:off #}
+    {{
+        onsHero({
+            "topic": hero_topic,
+            "title": page.title,
+            "variants": 'grey',
+            "officialStatisticsBadge": page.is_accredited,
+            "wide": true,
+            "detailsColumns": 12,
+            "censusLogo": page.is_census,
+            "descriptionList": {
+                "descriptionListLabel": "",
+                "termCol": "6",
+                "descriptionCol": "6",
+                "itemsList": [
+                    release_date_html,
+                    next_release_html,
+                ]
+            },
+            "breadcrumbs": {
+                "ariaLabel": breadcrumbs_aria_label,
+                "itemsList": breadcrumbs(page),
+            }
+        })
+    }}
+    {# fmt:on #}
 
-        {% if page.is_census %}
-            <div class="ons-container">
-                {% with census_logo_classes="ons-u-mt-2xs ons-u-mb-s ons-u-mt-s@m census-logo" %}
-                    {% include "templates/components/census/census-logo.html" %}
```

Comment:
question: do we still need the partial include now that this is part of the hero component?

Reply:
We're still using these components in the main body of the release calendar page: 

![image](https://github.com/user-attachments/assets/5a362c13-0032-45af-8f31-c21f447262dc)

so I'll leave them in for now until we update the content/template later